cmd/epp/main.go

```diff
@@ -172,6 +191,23 @@ func run() error {
 	datastore := datastore.NewDatastore(ctx, pmf)
 
 	scheduler := scheduling.NewScheduler(datastore)
+	if schedulerV2 == "true" {
+		queueConfig := scorer.QueueScorerConfig{
+			Weight: envutil.GetEnvInt("QUEUE_SCORE_WEIGHT", scorer.DefaultQueueScorerWeight, setupLog),
+		}
+		kvCacheConfig := scorer.KVCacheScorerConfig{
+			Weight: envutil.GetEnvInt("KV_CACHE_SCORE_WEIGHT", scorer.DefaultKVCacheScorerWeight, setupLog),
+		}
```

Comment:
weight of a scorer is not a configuration flag at the scorer level, but at the scheduler level.
the weights are relative to the weights of the other scorers (e.g, if I'll add my scorer with weight 100, would you still use default weight 2?).

I think the weights should stored somewhere else and not in the scorer itself.

Reply:
Right, I realize it's a bit over engineering to define a Config for each scorer. So I simply read the weight and add the scorer and its weight to the scheduler config.